sidebars.js

```diff
@@ -37,7 +37,7 @@ const sidebars =
       [
         {
           type: 'category',
-          label: 'Docs Site',
+          label: 'Docs',
```

Comment:
We need to specify what this documentation is about. Calling it **'docs'** is very counter-intuitive.

Reply:
Changes updated!